IPG-Automotive-UK/react-ui
# Migrate Wizard components to typescript

## Changes

* Migrated `Wizard`, `WizardContent`, `WizardSteps`, `WizardStep`, `WizardActions`, `NextButton`, `BackButton`, `CancelButton` to typescript.
* Turned off some unnecessary story controls. 

## UI/UX

No changes

## Testing notes

- Ensure component acts and behaves as before.
- Ensure docs are correctly parsed from the types
- Ensure types look correct

## Author checklist before assigning a reviewer
